packages/components/scripts/wrapper-generator/AbstractWrapperGenerator.ts

```diff
@@ -67,7 +67,9 @@ export abstract class AbstractWrapperGenerator {
   }
 
   private getComponentSubDir(component: TagName): string {
-    return this.shouldGenerateFolderPerComponent(component) ? this.getComponentFileName(component, true) : '';
+    return this.shouldGenerateFolderPerComponent(component)
+      ? path.parse(this.getComponentFileName(component)).name
```

Comment:
After seeing this change like 7 times, let's create a util called `stripFileExtension` in this class and use it everywhere.